practice/5_additional_topics/parsing_serialization_task/part5.py

```diff
@@ -0,0 +1,79 @@
+import json
+import os
+import sys
+import xml.etree.cElementTree as ET
+from xml.dom import minidom
+
+rootdir = "/Users/broczniok/Desktop/PYTHON-BASIC/practice/5_additional_topics/parsing_serialization_task/source_data"
```

Comment:
Use relative paths. I keep the project in an another folder. Your code is not working for me